cg/meta/workflow/mutant/mutant.py

```diff
@@ -249,3 +270,65 @@ def link_nanopore_fastq_for_sample(
         LOG.info(f"Concatenation in progress for sample {sample.internal_id}.")
         self.fastq_handler.concatenate(read_paths, concatenated_path)
         self.fastq_handler.remove_files(read_paths)
+
+    def run_qc_and_fail_analyses(self, dry_run: bool) -> None:
+        """Run qc check, report qc summaries on Trailblazer and fail analyses that fail QC."""
```

Comment:
Removed the QC status step.
If there is an exception when running QC or the QC fails, the status if set to fail to prevent the analysis from being stored and then uploaded.